pptx/package.py

```diff
@@ -74,7 +74,7 @@ def next_image_partname(self, ext):
         def first_available_image_idx():
             image_idxs = sorted([
                 part.partname.idx for part in self.iter_parts()
-                if part.partname.startswith('/ppt/media/image')
+                if part.partname.startswith('/ppt/media/image') and part.partname.idx is not None
```

Comment:
Ah found it, thanks.

So the problem is a bit different. These are the two image lines from `unzip -l template.pptx`:

    63877  06-01-16 03:32   ppt/media/image01.png
    94080  06-01-16 03:32   ppt/media/image00.png

Note the different convention for the part numbering. I expect this is an artifact of Google Slides, which is simply preserved by PowerPoint when there are no changes.

I think the fix is going to be enhancing PackURI.idx to recognize leading zeros on the part names.

I'll have a closer look ...